source/data/model/MeshLoader.cpp

```diff
@@ -7,90 +8,119 @@
 
 #include <radix/core/gl/OpenGL.hpp>
 
-#include <radix/env/Environment.hpp>
 #include <radix/Entity.hpp>
-#include <radix/core/gl/TightDataPacker.hpp>
 #include <radix/component/Transform.hpp>
+#include <radix/core/gl/TightDataPacker.hpp>
+#include <radix/env/Environment.hpp>
 
 namespace radix {
 
-std::map<std::string, Mesh> MeshLoader::meshCache = { };
+std::map<std::string, Mesh> MeshLoader::meshCache = {};
 
-Mesh& MeshLoader::getMesh(const std::string &path) {
+Mesh &MeshLoader::getMesh(const std::string &path) {
+  // check if mesh is cahced
   auto it = meshCache.find(path);
-  if (it != meshCache.end()) {
+  if (it != meshCache.end())
     return it->second;
-  }
-  std::string fpath = Environment::getDataDir() + "/meshes/" + path;
+
+  // get file Name
+  const std::string filePath = Environment::getDataDir() + "/meshes/" + path;
+  // create instance from Importer
   Assimp::Importer importer;
-  int flags = aiProcess_Triangulate | aiProcess_GenNormals | aiProcess_CalcTangentSpace;
-  const aiMesh *mesh = importer.ReadFile(fpath, flags)->mMeshes[0];
-  Mesh m = uploadMesh(mesh);
-  auto inserted = meshCache.insert(std::pair<std::string, Mesh>(path, m));
+  // flags to read mesh
+  // - aiProcess_Triangulate      : Triangulates all faces of all meshes.
+  // - aiProcess_GenNormals       : Generates normals for all faces of all meshes.
+  // - aiProcess_CalcTangentSpace : Calculates the tangents and bitangents for
+  //   the imported meshes.
+  // http://assimp.sourceforge.net/lib_html/postprocess_8h.html
+  unsigned int flags =
+      aiProcess_Triangulate | aiProcess_GenNormals | aiProcess_CalcTangentSpace;
+  // read mesh from hard to container
+  const auto container = importer.ReadFile(filePath, flags);
+  // get first mesh from container
+  const aiMesh *mesh = container->mMeshes[0];
+  // Upload mesh to GPU
+  Mesh meshObject = uploadMesh(mesh);
+
+  // Add mesh to cache
+  auto inserted = meshCache.insert(std::make_pair(path, meshObject));
   // Return reference to newly inserted Mesh
   return inserted.first->second;
```

Comment:
Comments should be to explain why you are doing something, not what you are doing. In the case of flags this is fine, but in the case of getting the filepath, etc, the comments aren't needed. If they are just to help you along with writing code that's fine, I do it too, but just make sure to remove them before pushing : )